src/app/globals.css

```
@@ -292,3 +299,160 @@ td {
+  white-space: pre;
+}
+
+.systemd-boot-prompt-line {
+  white-space: pre;
+  tab-size: 8;
+}
+
+.systemd-boot-line-kernel {
+  color: #969696;
+  white-space: pre-wrap;
+  word-break: break-word;
```

Stylelint (17.12.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)